modules/ROOT/pages/ios_app/preparation/provisioning_profiles.adoc

```diff
@@ -8,130 +8,42 @@
 
 == Overview
 
-Eight provisioning profiles need to be created:
+The next step is to create five provisioning profiles: either 5 *App Store* profiles if you want to distribute the App via the App Store, or 5 *Ad Hoc* profiles if you want to distribute it differently . All of these profiles must be stored on your Mac PC.
+Create a Provisioning Profile
 
-* 4 ad hoc and
-* 4 app store.
 
-You will email the four ad hoc profiles and your P12 certificate (which you will create after your provisioning profiles), to support@owncloud.com after building your branded app with the ownBrander app on {customer-owncloud-url}[customer.owncloud.com]. *Do not send us the app store profiles.* All eight of these profiles must be stored on your Mac PC.
+== Create Provisioning Profiles
 
-== Ad Hoc Provisioning Profiles
+Now you must create your Provisioning Profiles. Go to *Account > Profiles* and click the *plus button* to open the "Register a New Provisioning Profile" screen.
 
-=== First Profile
+image::ios_app/preparation/provisioning_profiles/profile-1.png[image, width=450]
 
-Go to menu:Provisioning Profiles[All], then click the btn:[plus button] at the top right to open the _Add iOS Provisioning Profile_ screen. Select btn:[Ad Hoc] and click btn:[Continue].
+image::ios_app/preparation/provisioning_profiles/profile-2.png[image, width=450]
 
-image::ios_app/preparation/provisioning_profiles/cert-35.png[image, width=450]
+If you want to distribute the App by yourself, select *Ad Hoc*. Otherwise, select *App Store*. The process is similar for both distributions.
```

Comment:
```suggestion
If you want to distribute the App by yourself, select btn:[Ad Hoc]. Otherwise, select btn:[App Store]. The process is similar for both distributions.
```